src/features/game/events/landExpansion/removeCollectible.test.ts

```diff
@@ -374,4 +388,48 @@ describe("removeCollectible", () => {
 
     expect(gameState.inventory).toStrictEqual({ Kuebiko: new Decimal(1) });
   });
+
+  it("burns all flower seeds if a Hungry Caterpillar is removed", () => {
```

Comment:
Nice work @eliasSFL. 

Just a note, when adding logic to events its good practice to do Test Driven Development. This means building a test that tests the functionality first and then implementing the logic to make the test pass. 

I have added the test for the functionality here. Next time you work on event logic, give it a try 😄 

Great job.